src/pages/dar_application/dar_application_new.css

```diff
@@ -0,0 +1,36 @@
+.dar-application-row {
+  clear: both;
+  margin-top: 15px;
```

Comment:
Since it's the margin-top of a row, I'm guessing it's being used to add space between rows. So the question is do we want the space between rows to grow as the page gets taller or not?

IMO I prefer the spacing to be fixed (or at least not grow beyond a max value), since, in my mind, making the window taller means I want to see more rows at once. Using rem, em, or % here would make the spacing grow if the parent (em, %) or root(rem) also grows on height change.